plugins/usertiming.js

```diff
@@ -47,6 +51,8 @@
  *     {"name":"mark2","startTime":200,"duration":0,"entryType":"mark"},
  *     {"name":"measure2","startTime":200,"duration":200,"entryType":"measure"}]
  *
+ * Uncompressed measures only send their duration in contrast to comrpressed measures, which also send their startTime.
```

Comment:
s/comrpressed/compressed/

Reply:
For your use-case, you're not using a measure's `startTime` for anything?  Are we sure other consumers won't want access to the `startTime`?

Reply:
If you want me to, I can integrate it, but we don't measure it, since we can interpret the corresponding marks.

Reply:
That's fine then, we can keep it at just `duration` for now and if someone else needs `startTime` we can add an option